charts/rancher-turtles/100.1.0+up0.5.0/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+annotations:
```

Comment:
For your referenec this is a good example of Chart.yaml on dev-v2.8

https://github.com/rancher/charts/blob/dev-v2.8/charts/rancher-istio/103.2.0%2Bup1.20.3/Chart.yaml